src/main/java/net/glowstone/util/Typeless.java

```diff
@@ -0,0 +1,12 @@
+package net.glowstone.util;
+
+public class Typeless<T> {
```

Comment:
Pretty sure I deleted that, must have accidentally restored it when restoring other commits.